test_conformance/integer_ops/test_abs.c

```diff
@@ -324,6 +326,10 @@ int test_integer_abs(cl_device_id device, cl_context context, cl_command_queue q
         log_info("Failed on %d types\n", fail_count);
         return -1;
     }
+    if(compile_count) {
+        log_info("Failed to compile %d kernels\n", compile_count);
+        return -1;
```

Comment:
We should start using  enum test_status. See testHarness.h

Reply:
I will try using it in places I modify.